core/koin-core/src/commonMain/kotlin/org/koin/mp/KoinPlatformTools.kt

```diff
@@ -12,6 +12,8 @@ expect object KoinPlatformTools {
     fun defaultLazyMode(): LazyThreadSafetyMode
     fun defaultLogger(level: Level = Level.INFO): Logger
     fun defaultContext(): KoinContext
-    fun <R> synchronized(lock: Any, block: () -> R): R
+    fun <R> synchronized(lock: SyncCapable, block: () -> R): R
     fun <K, V> safeHashMap(): MutableMap<K, V>
-}
\ No newline at end of file
+}
+
+expect open class SyncCapable()
```

Comment:
I was thinking of `Lockable` instead of `SyncCapable` 🤔

Reply:
Yeah, OK. I'll rename that.

Reply:
thanks